tests/integration/targets/wafv2/tasks/create_webacl.yml

```diff
@@ -1,151 +1,230 @@
 #######################
 ## Create web acl
 #######################
-- name: check_mode create web acl
-  wafv2_web_acl:
-    name: "{{ web_acl_name }}"
-    state: present
-    description: hallo eins
-    scope: REGIONAL
-    default_action: Allow
-    sampled_requests: no
-    cloudwatch_metrics: yes
-    metric_name: blub
-    rules:
-      - name: zwei
-        priority: 2
-        action: 
-          block: {}
-        visibility_config: 
-          sampled_requests_enabled: yes
-          cloud_watch_metrics_enabled: yes
-          metric_name: ddos
-        statement:
-          xss_match_statement:
-            field_to_match:
-              body: {}
-            text_transformations:
-              - type: NONE
-                priority: 0
-      - name: admin_protect
-        priority: 1
-        override_action: 
-          none: {}
-        visibility_config: 
-          sampled_requests_enabled: yes
-          cloud_watch_metrics_enabled: yes
-          metric_name: fsd
-        statement:
-          managed_rule_group_statement:
-            vendor_name: AWS
-            name: AWSManagedRulesAdminProtectionRuleSet
-    tags:
-      A: B
-      C: D
-  register: out
-  check_mode: yes
 
-- name: check_mode verify create
-  assert:
-    that:
-      - out is changed
+# Install newer botocore (then on regular CI) to support all wafv2 features used here
```

Comment:
This can be much simplified

See https://github.com/ansible-collections/community.aws/pull/917/files#diff-43fcf447e9c62db1fb87fb3bd2c772230846b4c7ee91a114732fe4cd8b1960baR5

Just use `meta/main.yml` like this

```yml
dependencies:
  - setup_remote_tmp_dir
  - role: setup_botocore_pip
    vars:
      boto3_version: "1.18.0"
      botocore_version: "1.21.0"
```

and all what's need in the `create_webacl.yml is  

```yml
- name: Wrap test in virtualenv created above (use other python interpreter)
  vars:
    ansible_python_interpreter: "{{ botocore_virtualenv_interpreter }}"
  block:
    - name: check_mode create web acl
...
```

Reply:
Good to know :tada: